mixins/github-actions-iam-role/github-actions-iam-role.mixin.tf

```diff
@@ -48,6 +48,8 @@ module "gha_assume_role" {
   trusted_github_repos = var.github_actions_allowed_repos
 
   context = module.gha_role_name.context
+
+  stage = "auto"
```

Comment:
can this be a variable instead with default as "auto"? a few customers don't have the same stage names